src/main/java/com/wcc/platform/controller/CmsController.java

```diff
@@ -0,0 +1,26 @@
+package com.wcc.platform.controller;
+
+import com.wcc.platform.domain.pages.TeamPage;
+import com.wcc.platform.service.CmsService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/cms/v1/")
+
+public class CmsController {
+
+    private final CmsService cmsService;
+
+    @Autowired
+    public CmsController(CmsService service) {
+        this.cmsService = service;
+    }
+
+    @GetMapping("/team")
+    public TeamPage getTeamPage() {
+        return cmsService.getTeam();
```

Comment:
Should we use ResponseEntity class to return the response ? It can be more flexible and consistent in terms of return codes.